secboot/secboot.go

```diff
@@ -35,6 +36,21 @@ var (
 	devDiskByLabelDir = "/dev/disk/by-label"
 )
 
+type SealKeyParams struct {
+	// The set of EFI binary load paths for the current device configuration
+	EFILoadChains [][]string
+	// The kernel command line
+	KernelCmdlines []string
+	// The snap model
+	Models []*asserts.Model
```

Comment:
each model might have some kernel command lines and each might have some efi chains, no? it's a matrix of all models x all efi chains x command lines

Reply:
Changing the way we add profiles to a more structured way (so we can specify command lines and EFI load chains corresponding to specific snap models).